### PR TITLE
#167735029 Setup swagger documentation

### PR DESCRIPTION
**Pull request by prieste**
[starts #167735029]

**What does this PR do?**

 Setup swagger documentation

  **Description of Task to be completed?** 

 Set up documentation for available routes

  **How should this be manually tested?**

-  Start server using 'npm run server'
- navigate to 'localhost:3000/docs' on your browser

  **Any background context you want to provide?**

 N/A

  **What are the relevant stories?**

  #167735029 

  **Screenshots (if appropriate)**

 N/A

  **Questions:**

N/A

### CODE REVIEW

File: src/routes/index.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
 const router = require("express").Router();
+const swaggerUi = require('swagger-ui-express');
+const YAML = require('yamljs');
+
+const swaggerDocument = YAML.load(`${process.cwd()}/src/docs/docs.yaml`);
```
Comment (Annmary12) [inline src/routes/index.js:9]: @prieste, can you convert line 1 to 3  to make use of import.

File: src/routes/index.js [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+import { Router } from 'express';
+import swaggerUi from 'swagger-ui-express';
+import YAML from 'yamljs';
+
+const router = Router();
+
+const swaggerDocument = YAML.load(`${process.cwd()}/src/docs/docs.yaml`);
+
+router.use('/api', require('./api'));
+
```
Comment (Annmary12) [inline src/routes/index.js:12]: @prieste can you also change line 9 to es6(make use import to import the api).

Comment (prieste) [reply src/routes/index.js:12]: I have made the necessary changes

**Comment by Annmary12** [comment]
@prieste , Resolve conflict on your PR.

**Review by prieste — COMMENTED** [review]
> @prieste , Resolve conflict on your PR.

I've tended to the conflicts please review

**Review by Annmary12 — APPROVED** [review]

**Pull request merged**